common-docs/python/types.md

```diff
@@ -0,0 +1,188 @@
+# Types
```

Comment:
I would actually leave out this whole page on Types. We try our best to infer types as much as possible and if there are common cases that still need them we're likely to improve our inference in the future. I think this section will just confuse folks, especially those more familiar with Python.

If we find that omitting this is causing too much confusion, we can add some version back in the future.
FYI @abchatra